client/pokemon/pokemon-icon.view.html

```diff
@@ -0,0 +1,4 @@
+<div class="icon-container" ng-init="pokemon.parseCardViewProps()" flex
+  ng-class="['spr-box', pokemon.spriteClass, {'pokemon-egg': pokemon.data.isEgg}]" >
+  <span ng-if="::pokemon.data.isEgg" class="spr-box spr-box-egg"></span>
```

Comment:
I haven't actually, that's a good point I tested how it looked (by changing the data locally) but not from an actual egg. I shall do that today.